fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/api/resource/conditions/v1/ResourceConditions.java

```diff
@@ -122,6 +122,7 @@ public static Predicate<JsonObject> get(Identifier identifier) {
 	 * <p>If an exception is thrown during condition testing, it will be caught and logged, and false will be returned.
 	 */
 	public static boolean objectMatchesConditions(JsonObject object) {
+		if (JsonHelper.hasArray(object, "conditions")) return true;
```

Comment:
I am not 100% sure whether there is a better way to reference `"conditions"`. I couldn't find a constant which contains this string in Forge's source code, so I hardcoded it here for now... Is there something?